scripts/setup-helper-functions.sh

```diff
@@ -140,6 +142,37 @@ function cmake_install {
     -DCMAKE_CXX_FLAGS="$COMPILER_FLAGS" \
     -DBUILD_TESTING=OFF \
     "$@"
-  ninja -C "${BINARY_DIR}" install
+
+  echo $BINARY_DIR
+}
+
+# install a prebuilt project
+function ninja_install {
+  local BINARY_DIR=_build
+  ninja-build -C "${BINARY_DIR}" install
+}
+
+# install a prebuilt project with elevated privileges
+# This is useful for systems where /usr/ is not writable
+function sudo_ninja_install {
+  local BINARY_DIR=_build
+  sudo ninja-build -C "${BINARY_DIR}" install
```

Comment:
ups, this is the wrong binary. I'll revert those changes and move the logic to the new file

Reply:
Done